tests/js/helpers/helpers-tests.js

```diff
@@ -0,0 +1,52 @@
+/*
+
+  We provide some "helper" grades to assist other people in test routers and other components that run in conjunction
+  with an instance of `gpii.express`.  These tests specifically exercise those test components.
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("./includes.js");
+
+var path    = require("path");
+var viewDir = path.resolve(__dirname, "./views");
```

Comment:
Not required for this pull to get in, but you may be interested in time to move over to the module interpolation scheme supported by "new kettle" that would allow you to express this as "%gpii-express/tests/js/helpers/views" in the configuration directly. This manual use of the "path" module was one of the evils in "old kettle" that prevented our configuration from being proper configuration.